portfolio/src/main/webapp/css/master.css

```diff
@@ -1,3 +1,8 @@
+#fact-container {
+  height: 75px;
+  overflow-y: auto !important;
```

Comment:
Maybe a comment explaining why !important here is necessary would be good.

Reply:
Good point. !important isn't necessary here, and even though I realized that I accidentally did not save the file before staging! Will update.